addon/pods/components/rui-error-template/component.js

```diff
@@ -5,6 +5,12 @@ export default Ember.Component.extend({
   classNames: ['rui-error-template'],
   layout,
 
+  externalLink: Ember.computed(function() {
+    let link = this.get('link')
+
+    return link.indexOf('http') >= 0 ? true : false
```

Comment:
no...just looking it up by whether it is a link to an http: leading url.  Relative urls will start with slashes or text.